modules/ossm-deploying-bookinfo-application-istio-ambient-mode.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/install/ossm-istio-ambient-mode.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-bookinfo-application-istio-ambient-mode_{context}"]
+= Deploying the Bookinfo application in Istio ambient mode
+
+You can deploy the `bookinfo` sample application in {istio} ambient mode without sidecar injection by using the `ZTunnel` proxy. For more information on `bookinfo` application, see "About the Bookinfo application".
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later.
+* You are logged in to the {ocp-product-title} either through the web console as a user with the `cluster-admin` role, or with the `oc login` command, depending on the installation method.
+* You have installed the {SMProductName} Operator, created the {istio} resource, and the Operator has deployed {istio}.
+* You have created an `IstioCNI` resource, and the Operator has deployed the necessary `IstioCNI` pods.
+* You have created a `Ztunnel` resource, and the Operator has deployed the necessary `Ztunnel` pods.
+
+.Procedure
+
+. Create the `bookinfo` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace bookinfo
+----
+
+. Add the `istio-discovery=enabled` label to the `bookinfo` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-discovery=enabled
+----
+
+. Apply the `bookinfo` YAML file to deploy the `bookinfo` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.26/samples/bookinfo/platform/kube/bookinfo.yaml
```

Comment:
@MaxBab We shouldn't reference community Github artifacts in istio/istio for product documentation. Is there an equivalent in openshift-service-mesh/istio we can use? If not, we'll need to create it or this will be considered a bug.

Reply:
I think it's ok to have a separate bookinfo example in the docs for ambient - it is different enough. We may also want to take the example further. As long as we replace these links (which use containers hosted on Red Hat's quay rahter than upstream's containers on Dockerhub), we should be fine and I will approve this PR.